src/portal/src/lib/components/create-edit-endpoint/create-edit-endpoint.component.ts

```diff
@@ -477,4 +477,8 @@ export class CreateEditEndpointComponent
     }
     return changes;
   }
+
```

Comment:
#11423  has not been repaired (operation is as follows)
1. create a endpoint
2. input name
3. delete name (prompt is required)
4. Click cancel button
5 create a endpoint
![image](https://user-images.githubusercontent.com/29826789/84231599-143c7b80-ab21-11ea-9226-ce7ab9efb860.png)

Reply:
Fixed already. 
![image](https://user-images.githubusercontent.com/30999793/84235923-51a50700-ab29-11ea-9111-21fd020b5180.png)